src/pydiode/send.py

```diff
@@ -54,10 +61,13 @@ async def read(self):
             # redirection.
             # https://stackoverflow.com/a/71627449
             if not self.stream_reader:
-                loop = asyncio.get_event_loop()
-                self.stream_reader = asyncio.StreamReader()
-                protocol = asyncio.StreamReaderProtocol(self.stream_reader)
-                await loop.connect_read_pipe(lambda: protocol, sys.stdin.buffer)
+                if platform == "win32":
+                    return await asyncio.to_thread(sys.stdin.buffer.read, self.chunk_max_data_bytes)
```

Comment:
I'm still a little confused about why the Linux branch of the logic has to wait for the read pipe to connect, but here we just await the read. Would the logic you wrote for Windows also work directly on Linux?